core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -397,17 +406,32 @@ angular.module('oppia').component('contributionsAndReview', {
         ctrl.reviewTabs = [];
         ctrl.contributionTabs = [
           {
-            suggestionType: SUGGESTION_TYPE_QUESTION,
+            suggestionType: ctrl.SUGGESTION_TYPE_QUESTION,
             text: 'Questions',
             enabled: false
           },
           {
-            suggestionType: SUGGESTION_TYPE_TRANSLATE,
+            suggestionType: ctrl.SUGGESTION_TYPE_TRANSLATE,
             text: 'Translations',
             enabled: true
           }
         ];
 
+        var prevSelectedTopicName = (
```

Comment:
Declare as `const`?

Reply:
removed all the code, we don't need this portion now that we moved the topic selector.

Reply:
I'm still seeing the topic selection logic here?

Reply:
Er sorry removed once again